bindings/nodejs/lib/index.ts

```diff
@@ -41,42 +52,95 @@ export * from './wallet';
 export * from './logger';
 
 // For future reference to see what we return from rust as a serialized string
-export type Result = {
-    // "error" | "panic", or other binding "Response" enum name, we consider "ok".
+export type Result = OkResult | NotOkResult;
```

Comment:
Can we just do this instead
```suggestion
export type Result = OkResult | ErrorResult | PanicResult;
```

Reply:
Down below i would then need to do err = err as (ErrorResult | PanicResult);
Not too pretty either then

Reply:
I prefer that tbh

Reply:
Done-